### DISCUSSION TITLE
STM32F407+LAN8720 Connect multiple servers

### DISCUSSION THREAD

**Discussion by banbeiduyao** [stm32duino/Arduino_Core_STM32#2521] [Q&A]
After testing, the maximum number of connections can be up to 10. Where can I modify the maximum number of connections?
Library link[https://github.com/khoih-prog/EthernetWebServer_STM32](https://github.com/khoih-prog/EthernetWebServer_STM32)

**Comment by fpistm** [comment]
Hi @banbeiduyao 
This library is not supported by this GitHub organization.
As it is archived, you can try to ask on the forum.
